src/mbed_tools/cli/main.py

```diff
@@ -37,22 +37,12 @@ def invoke(self, context: click.Context) -> None:
 
 
 def print_version(context: click.Context, param: Union[click.Option, click.Parameter], value: bool) -> Any:
-    """A click callback which prints the versions of all Mbed Python packages."""
+    """A click callback that prints the versions of mbed-tools."""
```

Comment:
Should this be "version" instead of "versions"? Does mbed-tools have multiple versions?

Reply:
It was printing the versions for all installed `mbed-` packages. I missed that, will change it.